src/pieces/commandManager.ts

```diff
@@ -14,6 +14,18 @@ async function register(bot: Client): Promise<void> {
 		bot.emit('error', error);
 	}
 
+	bot.on('messageCreate', async msg => {
+		if (msg.content.substring(0, 8).toLowerCase().includes('s;enroll')) {
```

Comment:
```suggestion
		if (msg.content.substring(0, 2).toLowerCase().includes('s;')) {
```
Because we're looking for all legacy commands now, not just enroll.